lib/iris/tests/runner/_runner.py

```diff
@@ -163,7 +163,7 @@ def run(self):
 
         args = ['', None, '--processes=%s' % n_processors,
                 '--verbosity=2', regexp_pat,
-                '--process-timeout=600']
+                '--process-timeout=180']
```

Comment:
This number sounds suspiciously like the default image tolerance from the image tests... Still, we can change the value as we need, and always put some effort into fixing it in a year or two :wink: 

And it's great to get the travis tests running faster! Good stuff @bjlittle :+1: